.github/workflows/java-pr.yml

```diff
@@ -74,3 +74,24 @@ jobs:
         run: ./cicd/run-unit-tests --changed-files="${{ steps.setup-env.outputs.changed-files }}"
       - name: Cleanup Java Environment
         uses: ./.github/actions/cleanup-java-env
+  java_integration_tests:
+    name: Integration Tests
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@230611dbd0eb52da1e1f4f7bc8bb0c3a339fc8b7
+      - name: Setup Environment
+        id: setup-env
+        uses: ./.github/actions/setup-env
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      - name: Build with Maven
+        run: mvn install -f pom.xml -pl .,it,syndeo-template,v1 -DskipTests
+      - name: Run integration tests with Maven
```

Comment:
I think at some point we need to make sure to authorize with WIF to access to the necessary resources: https://github.com/google-github-actions/auth